configuration.packages.json

```diff
@@ -72,22 +72,22 @@
     },
     "EdFi.Database.Admin.PostgreSQL": {
       "PackageName": "EdFi.Database.Admin.PostgreSQL",
-      "PackageVersion": "6.2.469",
+      "PackageVersion": "6.2.536",
       "PackageSource": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
     },
     "EdFi.Database.Admin": {
       "PackageName": "EdFi.Database.Admin",
-      "PackageVersion": "6.2.469",
+      "PackageVersion": "6.2.536",
       "PackageSource": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
     },
     "EdFi.Database.Security": {
       "PackageName": "EdFi.Database.Security",
-      "PackageVersion": "6.2.502",
+      "PackageVersion": "6.2.568",
```

Comment:
Does this package need an update? This is the database backup file used in the docker deployments.